app/src/userpages/components/StreamPage/Show/index.jsx

```diff
@@ -165,7 +167,14 @@ export class StreamShowView extends Component<Props, State> {
     }
 
     render() {
-        const { editedStream, cancel, currentUser, authApiKeyId } = this.props
+        const {
+            editedStream,
+            cancel,
+            currentUser,
+            authApiKeyId,
+            permissions,
+        } = this.props
+        const hasWritePermission = (permissions && permissions.some((p) => p === 'write')) || false
```

Comment:
I can't recall - do you remember if `share` permission could have the same properties as a `write` permission @tumppi ?

Reply:
I believe you can have `share` and `write` permissions separately